components/dash-core-components/tests/integration/dropdown/test_dropdown_shorthands.py

```diff
@@ -0,0 +1,52 @@
+from dash import Dash, dcc, html
+
+
+def test_ddsh001_dropdown_shorthand_properties(dash_dcc):
+    app = Dash(__name__)
+
+    TEST_OPTIONS_N_VALUES = [
+        [["a", "b", "c"]],
+        [["a", "b", "c"], "b"],
+        [["a", 3, "c"]],
+        [["a", 3, "c"], 3],
+        [["a", True, "c"]],
+        [["a", True, "c"], True],
+        [["a", 3, "c", True, False]],
+        [["a", 3, "c", True, False], False],
+        # {`value1`: `label1`, `value2`, `label2`, ...}
+        [{"one": "One", "two": "Two", "three": "Three"}],
+        [{"one": "One", "two": "Two", "three": "Three"}, "two"],
+        [{"one": 1, "two": 2, "three": False}],
+        [{"one": 1, "two": 2, "three": False}, "three"],
+        [{"one": 1, "two": True, "three": 3}],
+        [{"one": 1, "two": True, "three": 3}, "two"],
+        # original options format
+        [
+            [
+                {"label": "one", "value": 1},
+                {"label": "two", "value": True},
+                {"label": "three", "value": 3},
+            ]
+        ],
+        [
+            [
+                {"label": "one", "value": 1},
+                {"label": "two", "value": True},
+                {"label": "three", "value": 3},
+            ],
+            True,
+        ],
+    ]
+
+    layout = []
+    for definition in TEST_OPTIONS_N_VALUES:
+        (option, value) = definition if len(definition) > 1 else [definition[0], None]
+        layout.extend(
+            [html.Div(f"Options={option}, Value={value}"), dcc.Dropdown(option, value)]
```

Comment:
Could we render `RadioItems` and `Checklist` here too? Just to lock down the behavior of all of the components that accept these new properties